pkg/addon/operator_run.go

```diff
@@ -51,6 +51,16 @@ func InferOperatorRunType(ctx context.Context, client k8sclient.Client, installa
 
 	if subscription != nil {
 		return runTypesBySubscription[subscription.Name], nil
+	} else {
+		// If subscription is nil, it could be because the CPaaS build has put a non deterministic name on it.
+		// This code may be temporary if CPaaS changes to deterministic Subscription names
+		exists, err := RhoamCPaaSSubscriptionExists(ctx, client, installation.Namespace)
+		if err != nil {
+			return "", err
+		}
+		if exists {
+			return AddonRunType, nil
```

Comment:
Is it addon in case of install type that CVP uses? I thought CVP uses OLM

Reply:
@sergioifg94 because this is CVP and not added through ManagedTenants maybe this should be OLM?

Reply:
@psturc We won't need to set the type with the new approach. Can you have another quick review of this?